jobs/devex/jenkins-bump-version/Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env groovy
+
+properties(
+    [
+        [ $class : 'ParametersDefinitionProperty', parameterDefinitions: [
+                [
+                    name: 'JENKINS_VERSION',
+                    description: 'Minimum required Jenkins version',
```

Comment:
Seems like the wrong description. I also like to put an example valid value in the description.

Reply:
✔️